packages/wallet-sdk/src/connection/WalletLinkConnection.ts

```diff
@@ -189,7 +202,7 @@ export class WalletLinkConnection {
             sessionIdHash: Session.hash(sessionId),
             metadata_keys: msg && msg.metadata ? Object.keys(msg.metadata) : undefined,
           });
-          this.sessionConfigSubject.next({
+          this.sessionConfigListner?.({
```

Comment:
sessionConfigSubject's been removed.
we can directly call this.sessionConfigListner? instead